clar2wasm/tests/wasm-generation/control_flow.rs

```diff
@@ -62,3 +96,22 @@ proptest! {
         );
     }
 }
+
+proptest! {
+    #![proptest_config(super::runtime_config())]
+
+    #[test]
+    fn begin((expr, expected_val, is_response_intermediary) in begin_strategy()) {
+        let mut expected_val:Result<Option<Value>, ()> = Ok(Some(expected_val.into()));
+
+        if is_response_intermediary{
+            expected_val=Err(());
+        }
```

Comment:
No need for a mutable variable

```suggestion
        let expected_val = if is_response_intermediary {
            Err(())
        } else {
            Ok(Some(expected_val.into()))
        };
```

Reply:
Fixed. I wanted to avoid `else`. What could be the main reason to avoid `mut` here? (I know immutable variables are preferred to avoid potential future problems, but it's a simple test)

Reply:
Why do you want to avoid `else`? 
I believe it's cleaner to have the expected result clearly stating that it's either ... or ... than stating it's something, and then later saying "oh btw, if something, then it actually would be ..."

Reply:
I get your point. Blocks irritate me. It's just personal choice.